cmd/root.go

```diff
@@ -169,6 +169,8 @@ When this flag is not set, there is no limit.`)
 to close after receiving a TERM signal. The proxy will shut
 down when the number of open connections reaches 0 or when
 the maximum time has passed. Defaults to 0s.`)
+	cmd.PersistentFlags().StringVar(&c.conf.Host, "host", "https://alloydb.googleapis.com/v1beta",
+		"When set, the proxy uses this host as the base API path. Example: https://alloydb.googleapis.com/v1beta")
```

Comment:
You can delete "Example: https://alloydb.googleapis.com/v1beta"  since the output will include the default already.